arch/arm/src/rtl8720c/amebaZ.c

```diff
@@ -0,0 +1,47 @@
+/****************************************************************************
+ * arch/arm/src/rtl8720c/amebaZ.c
```

Comment:
Don't use lower case mixed with upper case file names. Please rename the file to amebaz.c

Reply:
ok，i'll modify it